realm/realm-library/src/androidTest/java/io/realm/RealmObjectTests.java

```diff
@@ -207,6 +211,21 @@ public void deleteFromRealm_twiceThrows() {
         }
     }
 
+    @Test
+    public void deleteFromRealm_throwOnNullRow() {
```

Comment:
throwOnNullRow should either be `throwOnStandaloneObject` or `throwOnUnmanagedObject` IMO